ci/check-crates.sh

```diff
@@ -31,6 +31,9 @@ printf "%s\n" "${files[@]}"
 error_count=0
 for file in "${files[@]}"; do
   read -r crate_name package_publish workspace < <(toml get "$file" . | jq -r '(.package.name | tostring)+" "+(.package.publish | tostring)+" "+(.workspace | tostring)')
+  if [ "$crate" == "solana-bundle" ]; then
```

Comment:
what this?

Reply:
this script checks if the crate is on crates.io otherwise ci fails